public/app/config/components.js

```diff
@@ -8,6 +8,79 @@
         function (ko) {
             return function () {
                 ko.components.register(
+                    'collections/library/detail',
+                    { template: { require: 'text!ui/components/collections/library/detail.html' } }
```

Comment:
When a component doesn't specify a `viewModel`, only the `template`, then it is bound to whatever is passed as the `parameters`.